### PR TITLE
 desktop view for pr review

### PR DESCRIPTION
**Pull request by supreeti**
Add html and Css file for desktop view.

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -607,30 +608,40 @@ label {
...
+  }
+
+  .social-media {
+    display: block;
     justify-content: center;
+    width: 90%;
+    margin: auto;
+    padding-left: 120px;
+    padding-right: 120px;
   }
```
Comment (bienvenuushindi) [inline index.css:646]: :red_circle: [OPTIONAL] Please make sure your desktop version doesn't affect the mobile version of your portfolio. As for now, the mobile version has been affected immensely. Kindly fix this to have a professional portfolio. I think the big issue is with your about section The language, and skills framework section should move beneath the about block on the mobile version.

Peek 2023-01-21 23-17

File: index.css [hunk 2]
```diff
@@ -711,6 +722,7 @@ label {
     font-weight: 400;
     text-align: justify;
     justify-content: center;
+    padding-right: 90px;
```
Comment (bienvenuushindi) [inline index.css:725]: :red_circle: [REQUESTED] As requested by the previous reviewer, Kindly make sure you fix the style of this paragraph to match the Figma Design. I think you can use pixels or rem for the font-size property to fix this

Your Design
Screenshot from 2023-01-21 23-20-02


Figma Design
Screenshot from 2023-01-21 23-19-45

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
## Hi @bienvenuushindi,

Good job so far!

## Highlights!
- GitFlow ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 


Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [ ]  Please for future review Kindly make sure you keep the same pull request, (in this case it is #9 ) until it gets approved. when requesting for a re-review don't open a  new pull request use the previous one 

_Check the comments under the review and here 👇._


## Optional suggestions
- [ ] Kindly consider the optional comment left by the previous reviewer.
_Check the comments under the review and here 👇._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**